fastlane/lib/fastlane/actions/get_info_plist_value.rb

```diff
@@ -10,6 +10,12 @@ def self.run(params)
 
         begin
           path = File.expand_path(params[:path])
+          
+          unless File.exist?(path)
+            UI.error("Unable to find plist file at '#{path}'")
+            return
+          end
+          
           plist = Plist.parse_xml(path)
```

Comment:
Thanks for catching this. I usually do this:

```Ruby
plist = File.open(path) { |f| Plist.parse_xml(f) }
```

The problem is actually here: https://github.com/patsplat/plist/blob/master/lib/plist/parser.rb#L66. If the file does not exist, the input is assumed to be XML. Parsing fails because it's a path, and the method returns nil.

I prefer the approach above because `File.open` raises `Errno::ENOENT` in this case, so you get consistent exception handling instead of the custom error message for this one case, and it makes the code simpler.